graphql_api/types/comparison/comparison.graphql

```diff
@@ -14,6 +14,7 @@ type Comparison {
   hasDifferentNumberOfHeadAndBaseReports: Boolean!
   flagComparisonsCount: Int!
   componentComparisonsCount: Int!
+  impactedFilesNew(filters: ImpactedFilesFilters): ImpactedFilesResult!
```

Comment:
this would be `impactedFilesNew(filters: ImpactedFilesFilters): [ImpactedFilesResult!]` since it's returning a list. 

Idk if this is in scope for this PR but we might want to also create a new version of a singular impacted file (see `impactedFile`)